lisa/notifiers/junit.py

```diff
@@ -141,7 +141,9 @@ def _test_case_running(self, message: TestResultMessage) -> None:
 
     # Test case completed message.
     def _test_case_completed(self, message: TestResultMessage) -> None:
-        testsuite_info = self._testsuites_info[message.suite_full_name]
+        testsuite_info = self._testsuites_info.get(message.suite_full_name)
```

Comment:
When will it happen?

Reply:
When there is an error during Environment initialization. So, the test case never enters the running state.